editor/src/components/canvas/canvas-strategies/strategies/group-like-helpers.ts

```diff
@@ -0,0 +1,61 @@
+import { MetadataUtils } from '../../../../core/model/element-metadata-utils'
+import * as EP from '../../../../core/shared/element-path'
+import { ElementInstanceMetadataMap } from '../../../../core/shared/element-template'
+import { ElementPath } from '../../../../core/shared/project-file-types'
+import { AllElementProps } from '../../../editor/store/editor-state'
+import {
+  getTargetPathsFromInteractionTarget,
+  InteractionCanvasState,
+} from '../canvas-strategy-types'
+import { getDragTargets } from './shared-move-strategies-helpers'
+
+export function retargetStrategyToChildrenOfGroupLike(
+  canvasState: InteractionCanvasState,
+): Array<ElementPath> {
+  const targets = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+
+  const targetsWithoutDescedants = getDragTargets(targets)
+
+  return replaceGroupLikePathsWithTheirChildrenRecursive(
+    canvasState.startingMetadata,
+    canvasState.startingAllElementProps,
+    targetsWithoutDescedants,
+  )
+}
+
+function replaceGroupLikePathsWithTheirChildrenRecursive(
+  metadata: ElementInstanceMetadataMap,
+  allElementProps: AllElementProps,
+  paths: Array<ElementPath>,
+): Array<ElementPath> {
+  return paths.flatMap((path) => {
+    const elementIsGroupLike = treatElementAsGroupLike(metadata, allElementProps, path)
+
+    if (elementIsGroupLike) {
+      const children = MetadataUtils.getChildrenPathsUnordered(metadata, path) // I think it's fine to get the unordered children here?
```

Comment:
As long as we're sticking with absolute positioning only the order shouldn't matter here